### PR TITLE
docs(root): added pt_PT Readme file

### PR DESCRIPTION
**Pull request by jeanpierrecarvalho**
**Why?**

Added a pt_PT Readme.md file because:
- pt readme is written in Portuguese of Brazil, in Portugal some words have different meanings
- some terms are incorrect
- it's outdated

**What?**
- Add Readme.pt_PT.md

**How?**
- Add Readme.pt_PT.md on the root of the project
- Translate the english version to portugue of Portugal

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. README.md
1. README.pt_PT.md

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 137d13fdf475293ab451bb8a2e41e8f4fc9ab127
</p>

File: README.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ You can clone the entire repository [as per usual](https://docs.github.com/en/gi
 
 For a more manageable clone that includes _only_ the type packages relevant to you, you can use git's [`sparse-checkout`](https://git-scm.com/docs/git-sparse-checkout) and [`--filter`](https://git-scm.com/docs/git-rev-list#Documentation/git-rev-list.txt---filterltfilter-specgt) features. This will reduce clone time and improve git performance.
 
-> :warning: This requires minimum [git version 2.27.0](https://git-scm.com/downloads), which is likely newer than the default on most machines. More complicated procedures are available in older versions, but not covered by this guide.
+> ⚠️ This requires minimum [git version 2.27.0](https://git-scm.com/downloads), which is likely newer than the default on most machines. More complicated procedures are available in older versions, but not covered by this guide.
```
Comment (jakebailey) [inline README.md:160]: Please revert all changes to this file that are unrelated to the new translation.

**Review by jakebailey — CHANGES_REQUESTED** [review]
